administrator/language/en-GB/com_joomlaupdate.ini

```
@@ -3,18 +3,30 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
+COM_JOOMLAUPDATE_AUTOUPDATE_REGISTER_SUCCESS="Successful registered to autoupdate service"
+COM_JOOMLAUPDATE_AUTOUPDATE_REGISTER_ERROR="Error while registering to autoupdate service: %s (%d)"
+COM_JOOMLAUPDATE_AUTOUPDATE_UNREGISTER_SUCCESS="Successful unregistered from autoupdate service"
+COM_JOOMLAUPDATE_AUTOUPDATE_UNREGISTER_ERROR="Error while unregistering from autoupdate service: %s (%d)"
 COM_JOOMLAUPDATE_CAPTIVE_HEADLINE="Confirm your credentials"
 COM_JOOMLAUPDATE_CHECKED_UPDATES="Checked for updates."
+COM_JOOMLAUPDATE_CONFIG_AUTO_UPDATE_DISABLED_LABEL="<span class=\"fa-fw me-2 fa fa-info-circle\"></span> Automated updates are only supported for the \"default\" channel and \"Minimum Stability\" \"Stable\"."
+COM_JOOMLAUPDATE_CONFIG_AUTOMATED_UPDATES_LABEL="Automated updates"
+COM_JOOMLAUPDATE_CONFIG_AUTOUPDATE_DESC="Automatically update Joomla to the latest version when it is available."
+COM_JOOMLAUPDATE_CONFIG_AUTOUPDATE_LABEL="Automated Update"
 COM_JOOMLAUPDATE_CONFIG_BACKUPCHECK_DESC="Shows the checkbox to confirm you have taken a backup and you're ready to update in the final step before the update is actually applied."
 COM_JOOMLAUPDATE_CONFIG_BACKUPCHECK_LABEL="Confirm Backup Checkbox"
 COM_JOOMLAUPDATE_CONFIG_CUSTOMURL_LABEL="Custom URL"
 COM_JOOMLAUPDATE_CONFIG_SOURCES_DESC="Configure where Joomla gets its update information from."
 COM_JOOMLAUPDATE_CONFIG_SOURCES_LABEL="Update Source"
 COM_JOOMLAUPDATE_CONFIG_UPDATESOURCE_CUSTOM="Custom URL"
+COM_JOOMLAUPDATE_CONFIG_UPDATE_LAST_CHECK_LABEL="Last checked"
+COM_JOOMLAUPDATE_CONFIG_UPDATE_LAST_CHECK_DESC="The last time the automated update server could reach this site. If this date is too old, something is wrong with the automated update and you should re-initiate the process by saving the configuration."
```

its a bit clunky. what is "too old"?

```suggestion
COM_JOOMLAUPDATE_CONFIG_UPDATE_LAST_CHECK_DESC="The last time the automated update server could reach this site. If this date is too old, something is wrong with the automated update and you should re-initiate the process by re-saving the configuration."
```